packages/rum/test/e2e/standalone-html/app.e2e-spec.js

```diff
@@ -39,6 +39,18 @@ describe('standalone-html', function() {
     return allowSomeBrowserErrors(['timeout test error with a secret'])
   })
 
+  it('should run the async usecase', function() {
+    browser.url('/test/e2e/standalone-html/index-async.html').waitUntil(
```

Comment:
@jahtalab this cannot be chained with webdriver 5.0 version. have to do browser.waitUntil to make it work.

Reply:
Yeah, fixed it now!